src/uct/api/v2/uct_v2.h

```diff
@@ -478,6 +479,13 @@ typedef struct uct_md_mem_reg_params {
      * dmabuf region, then this field must be omitted or set to 0.
      */
     size_t                       dmabuf_offset;
+
+    /**
+     * Represents a pointer to the existing memory handle.
+     * Used to create a memory window is (with flag @ref UCT_MD_MEM_WINDOW)
+     * based on this original memh.
+     */
+    uct_mem_h                    memh;
```

Comment:
maybe we do not need both - the flag and memh?

Reply:
Right, flag can be removed